src/DynamoPackages/Package.cs

```diff
@@ -505,6 +509,10 @@ internal void UnmarkForUninstall(IPreferences prefs)
                 // Or maybe state should only be set when trying to Load the package...not here.
                 PackageState = PackageStates.Loaded;
             }
+            else
+            {
+                RaisePropertyChanged("EnableOldMarkedForUnistallState");
```

Comment:
this fixes a regression since the introduction of the debug mode EnableOldMarkedForUnistallState

Reply:
can nameof be used?